spec/unit/utils/encryptable_transfer_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe EncryptableTransfer do
+
+  let(:encryptable) { encryptables(:credentials1) }
+  let(:alice) { users(:alice) }
+  let(:bob) { users(:bob) }
+  let(:api_tux) { users(:api_tux) }
+  let(:bobs_private_key) { bob.decrypt_private_key('password') }
+  let(:team1) { teams(:team1) }
+
+  let(:encryptable_transfer) { described_class.new }
+
+  let(:encryptable_file) do
+    Encryptable::File.new(cleartext_file: 'fff', name: 'info.txt', content_type: 'text/plain')
+  end
+
+  context '#transfer' do
```

Comment:
for me it would be ok just having one context block ... transfer

and then test send and receive in the same block